Google.Api.Generator.Tests/ProtoTests/Lro/LroClient.cs

```diff
@@ -0,0 +1,135 @@
+﻿using gax = Google.Api.Gax;
+using gaxgrpc = Google.Api.Gax.Grpc;
+using lro = Google.LongRunning;
+using proto = Google.Protobuf;
+using grpccore = Grpc.Core;
+using grpcinter = Grpc.Core.Interceptors;
+using sys = System;
+using scg = System.Collections.Generic;
+using sco = System.Collections.ObjectModel;
+using st = System.Threading;
+using stt = System.Threading.Tasks;
+
+namespace Testing.Lro
+{
+    public abstract class LroClient
+    {
+        public lro::Operation<LroResponse, LroMetadata> SignatureMethod(Request request, gaxgrpc::CallSettings callSettings) => null;
```

Comment:
It looks like this is incomplete:

- Missing doc comments
- Missing virtual modifier
- Missing throwing implementation

Reply:
This line of code is not part of the test, so is deliberately missing anything that isn't strictly required for the code to compile.